Src/Plugins/Tests.Security/Validators/AzureDevOpsPersonalAccessTokenValidatorTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Security
+{
+    public class AzureDevOpsPersonalAccessTokenValidatorTests
+    {
+        [Theory]
```

Comment:
>Theory [](start = 9, length = 6)

No [Theory] utilization. :) Reflection is very slow for these. Just use unlined test data instead. There are some nice existing patterns for this in SARIF SDK.